packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -21,6 +21,13 @@ def main():
         if result.stderr.strip().endswith(b'File exists'):
             result.returncode = 0
         sys.exit(result.returncode)
+    if sys.argv[1] == 'iptables':
+        argv = ['-C' if arg in ['-A', '-I'] else arg for arg in sys.argv[1:]]
+        result = subprocess.run(argv)
+        if result.returncode == 0:
+            sys.exit(result.returncode)
+        else:
+            os.execvp(sys.argv[1], sys.argv[1:])
```

Comment:
This looks like: "if it didn't work the ` subprocess.run(argv)` way, then I don't know why really, and retry via `os.execvp(sys.argv[1], sys.argv[1:])`". Is that true? If that's true then I prefer to instead have one definite way of running this `iptables` command. If there is a real good explanation for this "retry" mechanism via a "different method" then I'd love to read about it.